### DISCUSSION TITLE
Order of Tags in Module Tags - Popular, according to Tags order.

### DISCUSSION THREAD

**Discussion by pwireless** [joomla/joomla-cms#47123] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

Order of Tags in Module Tags-Popular, allow only for 1. Title, 2. Number of items, 3. Random

### Describe the solution you'd like

Add 4. Tags order

In this case it should follow the order in the Tags component.

**Comment by pritam825** [comment]
![image](https://user-images.githubusercontent.com/54195688/141611428-0319e3a8-45e8-4cf0-a8a6-221b2e8afb19.png)
Are you talking about which tag in module, can you please post the screenshot of that

**Comment by richard67** [comment]
@pritam825 The description is clear. The popular tags module is a frontend module. You can find it in backend in modules manager when looking for module type "Tags-Popular". See here a description for J3 (it applies also to J4 regarding unctionality): https://docs.joomla.org/Help310:Extensions_Module_Manager_Tags_Popular . The author wants an additional ordering option to order by the ordering column as shown in backend in the tags component for the tags.

**Comment by richard67** [comment]
@pritam825 P.S.: Here is a description which shows how that module looks on frontend, https://www.collectiveray.com/joomla-tags . Just scroll down to section 2. "Joomla Tags module".

**Comment by richard67** [comment]
As this would be a new feature it would have to go into 4.1 if ready in time, otherwise 4.2.

**Comment by dgrammatiko** [comment]
Related https://github.com/joomla/joomla-cms/pull/19642
